packages/experimental/Divider/src/Divider.types.ts

```diff
@@ -0,0 +1,59 @@
+import { ViewStyle, ViewProps, ColorValue } from 'react-native';
+import { IconProps, IconSourcesType } from '@fluentui-react-native/icon';
+import { LayoutTokens } from '@fluentui-react-native/tokens';
+import { TextProps } from '@fluentui-react-native/text';
+
+export const divider = 'Divider';
+
+export const DividerInsetSizes = [0, 16, 56, 68, 72, 108] as const;
+export type DividerInsetSize = typeof DividerInsetSizes[number];
+
+export type DividerAlignment = 'start' | 'center' | 'end';
+export type DividerAppearance = 'default' | 'subtle' | 'brand' | 'strong';
+
+export interface DividerCoreProps {
+  color?: ColorValue;
+  insetSize?: DividerInsetSize;
+  vertical?: boolean;
+}
+
+export interface DividerProps extends DividerCoreProps {
+  alignContent?: DividerAlignment;
+  appearance?: DividerAppearance;
+  icon?: IconSourcesType;
+  text?: string;
```

Comment:
I noticed that the web component supports text as a child component in the format of <Divider>This is a Divider</Divider>. Did we diverge from this format deliberately?

Reply:
The main issue that came with passing text as a child was getting the text to be colored. I initially wrapped the text / icons in a wrapper slot and applied color styles there, but the colors failed to cascade down to the children.

Reply:
As of now, having props and feeding those to separate text and icon slots allows me to get styling correct, but I would love if there was a way to align more with the web api.

Reply:
For Text you can have the children be plain strings and can then pass the color down through Slots.label.

i.e. instead of passing in
<Divider><Text>Foo</Text></Divider>

pass in
<Divider>Foo</Divider>

Button has examples of this.
Unfortunately with icons you may need to have an icon prop and icon slot and diverge from web there.